packages/mdc-list/constants.js

```diff
@@ -18,14 +18,16 @@
 /** @enum {string} */
 const cssClasses = {
   LIST_ITEM_CLASS: 'mdc-list-item',
+  LIST_SELECTED_CLASS: 'mdc-list-item--selected',
 };
 
 /** @enum {string} */
 const strings = {
   ARIA_ORIENTATION: 'aria-orientation',
   ARIA_ORIENTATION_VERTICAL: 'vertical',
+  ARIA_SELECTED: 'aria-selected',
   FOCUSABLE_CHILD_ELEMENTS: 'button:not(:disabled), a',
-  ITEMS_SELECTOR: '.mdc-list-item',
+  ITEMS_SELECTOR: '.mdc-list-item:not(.mdc-list-item--disabled)',
```

Comment:
Why did you add the `:not(.mdc-list-item--disabled)` selector?

Reply:
Disabled items should not be receive focus, so the items selector should ignore them.

Reply:
Changing the key to `ENABLED_ITEMS_SELECTOR` is more semantically correct